VAMobile/src/utils/formattingUtils.ts

```diff
@@ -139,6 +139,95 @@ export const getEpochSecondsOfDate = (date: string): number => {
   ).toSeconds()
 }
 
+/**
+ * Format time in VA.gov style
+ *
+ * @param date - JavaScript Date object
+ * @param timeZone - Optional IANA timezone identifier
+ * @returns Time formatted as "8 AM" or "5 PM"
+ */
+const formatTimeVaStyle = (date: Date, timeZone?: string): string => {
+  let dt = DateTime.fromJSDate(date)
+  if (timeZone) {
+    dt = dt.setZone(timeZone)
+  }
+  // Format as hour followed by uppercase AM/PM (no minutes)
+  return dt.toFormat('h a')
+}
+
+/**
+ * Get timezone abbreviation from a date using native Intl API
+ * Uses native Date.toLocaleString() like web implementation, with GMT fallback workaround
+ *
+ * @param date - JavaScript Date object
+ * @param timeZone - Optional IANA timezone identifier
+ * @returns Timezone abbreviation (e.g., 'EDT', 'JST', 'PDT')
+ */
+const getTimezoneAbbr = (date: Date, timeZone?: string): string => {
+  const options: Intl.DateTimeFormatOptions = { timeZoneName: 'short' }
+  if (timeZone) {
+    options.timeZone = timeZone
+  }
+  let tzAbbr = date.toLocaleString('en-US', options).split(' ').pop() || ''
```

Comment:
Could this use the luxon DateTime 'ZZZZ' formatting token to reduce the amount of custom logic here? https://moment.github.io/luxon/#/formatting?id=table-of-tokens

Reply:
Hey @jmarchi! Updated in this commit - [0d26f08](https://github.com/department-of-veterans-affairs/va-mobile-app/pull/11760/commits/0d26f08e047c22d44b52ee4c550b5cfb342efd86). 

Also Mobile design team gave feedback to our designers to update content. Implemented the approved designs in this commit: [5ef1868](https://github.com/department-of-veterans-affairs/va-mobile-app/pull/11760/commits/5ef18684e4fc76d39ac0607d8fc8f0deb0cf1e38). [Here](https://www.figma.com/design/heeefL9b9INPwUqFj6dDtX/Claim-Status-Tool--Document-Upload-Date-Discrepancy-%7C-VA-Mobile?node-id=4152-8419&t=yZwh4rWj2wlj3g2c-4) is the link to the designs - there is a screenshot above as well. 

Also refined the testing in some subsequent commits! Ready for re-review!